### PR TITLE
Charts, docs, readability and other changes

### PR DESCRIPTION
**Pull request by khaosdoctor**

### CODE REVIEW

File: charts/operator/.helmignore [hunk 1]
```diff
@@ -0,0 +1,23 @@
+# Patterns to ignore when building packages.
```
Comment (tomkerkhove) [inline charts/operator/.helmignore]: Can we move this to `keda-http-operator` please?

File: charts/operator/Chart.yaml [hunk 2]
```diff
@@ -0,0 +1,23 @@
+apiVersion: v2
+name: operator
```
Comment (tomkerkhove) [inline charts/operator/Chart.yaml]:
```suggestion
name: keda-http-operator
```

File: charts/operator/Chart.yaml [hunk 3]
```diff
@@ -0,0 +1,23 @@
...
+
+# This is the chart version. This version number should be incremented each time you make changes
+# to the chart and its templates, including the app version.
+# Versions are expected to follow Semantic Versioning (https://semver.org/)
+version: 0.1.0
+
+# This is the version number of the application being deployed. This version number should be
+# incremented each time you make changes to the application. Versions are not expected to
+# follow Semantic Versioning. They should reflect the version the application is using.
+appVersion: 1.16.0
```
Comment (tomkerkhove) [inline charts/operator/Chart.yaml]: This feels more suitable maybe?
```suggestion
appVersion: 0.1.0
```

File: charts/operator/.helmignore (hunk 1 above)
Comment (khaosdoctor) [reply charts/operator/.helmignore]: Sure! I'll change it!

File: charts/operator/Chart.yaml (hunk 3 above)
Comment (khaosdoctor) [reply charts/operator/Chart.yaml]: Agree, I was trying to figure out where did helm took this version number

File: charts/operator/.helmignore (hunk 1 above)
Comment (khaosdoctor) [reply charts/operator/.helmignore]: Done

**Comment by khaosdoctor** [comment]
Added a few more changes:

- Create the interceptor and the external scaler for each new app (this can be changed later according to what @arschles told me about having a single external scaler)
- Add removal steps for newly created objects
- Removed environment variables that were not needed anymore
- Hardcoded exposed ports so we don't deal with complex port settings
- Added ingresses to the role because we'll need to create them
- Added a new status of "Terminating"
- Reordered the code and renamed files for readability

**Comment by arschles** [comment]
@khaosdoctor I'm gonna merge this now, and here are 3 follow-up issues that I'm going to start working on: https://github.com/kedacore/http-add-on/issues/5, https://github.com/kedacore/http-add-on/issues/4, https://github.com/kedacore/http-add-on/issues/3

**Review by arschles — APPROVED** [review]

**Pull request merged**